pallets/stellar-relay/src/lib.rs

```diff
@@ -117,6 +117,7 @@ pub mod pallet {
 		TransactionSetHashCreationFailed,
 		TransactionSetHashMismatch,
 		ValidatorLimitExceeded,
+		FailedToComputenonGenericTxSetContentHash,
```

Comment:
Nit: The 'n' for non should be capitalized
```
		FailedToComputeNonGenericTxSetContentHash,
```

Reply:
Done.